addons/binding/org.openhab.binding.plclogo/src/main/java/org/openhab/binding/plclogo/internal/PLCLogoClient.java

```diff
@@ -28,7 +28,7 @@
     private static final int MAX_RETRY_NUMBER = 10;
     private final Logger logger = LoggerFactory.getLogger(PLCLogoClient.class);
 
-    private String address;
+    private String address = "INVALID_IP";

```

Comment:
Hm. This is rather strange. Either make an `Optional<String>` or allow it to be null. But don't use arbitrary strings. Imagine somehow has setup a HOST file on his system an "INVALID_IP" actually points somewhere (very unlikely but possible).

Reply:
Leave it as is: It's used for logging only.

Reply:
I just thought I spotted a bug in line 63 and then noticed the different capitalization. Would it be possible that you add a "this.address" to that line or rename the field to "addressString" or so (might be more obvious that it is for logging only).
Thanks.

Reply:
Added comment and renamed variable to avoid confusion.